pvlib/irradiance.py

```diff
@@ -376,7 +376,7 @@ def total_irrad(surface_tilt, surface_azimuth,
     model = model.lower()
     if model == 'isotropic':
         sky = isotropic(surface_tilt, dhi)
-    elif model == 'klutcher':
+    elif model in ['klucher', 'klutcher']:
```

Comment:
This was already merged in #100 and then I rebased after the 0.2.2 release, so I didn't expect this to show up on the diff. Maybe I should have merged 0.2.2 into my branch instead of rebased.

The deprecation warning probably would have been nicer.